wasm/modtypes.py

```diff
@@ -90,12 +90,14 @@ def from_raw(self, struct, raw):
         from .decode import decode_bytecode
 
         offs = 0
+        offset = []
```

Comment:
Please rename this to something like `instructions` or `insns`, since it does not actually store offsets, but `Instruction` instances. This confused me for a second.

Reply:
Agree! Actually it stores ``expressions`` , which are sequences of instructions terminated by an 𝖾𝗇𝖽 marker (described here in [doc](https://webassembly.github.io/spec/core/syntax/instructions.html#syntax-expr).) Hmmm but looks like ``exprs`` is not as intuitive as ``instrs``, and anyway they are ``Instruction`` instances. Plus, ``end`` can be regarded as pseudo-instructions as well. So I agree with ``instrs``.

Thanks for your review! I made another commit to fix it.